changelog.yaml

```diff
@@ -0,0 +1,32 @@
+## ----------------------------------
+## Copy this template for new entries
+## ----------------------------------
+#    - type: [bug, enhancement, internal]
+#      impact: [incompatible, minor, patch]
+#      title: <title>
+#      description: >-
+#        <description>
+#      warning: >-
+#        <optional warning message>
+#      upgradeNotes: >-
+#        <optional upgrade guidelines>
+#      pullRequest: <pull request number>
+#      jiraIssue: <Jira issue number in CLOUDCISERVICE>
+
+- version: "NEXT"
+  date: TBD
+  changes:
+  - type: enhancement
+    impact: incompatible
+    title: Upgrade dependency to Tekton v0.14.3
+
+    description: >-
+      Build and run against Tekton v0.14.3.
+    warning: Requires Tekton v0.14.3.
+
+    upgradeNotes: >-
+      The target Kubernetes system must have Tekton v0.14.3 installed. Higher versions may also work.
+      In addition, Steward Jenkinsfile Runner Image version <a href="https://github.com/SAP/stewardci-jenkinsfilerunner-image/releases/tag/200921_6cc247f">200921_6cc247f</a> or later is required.
+      or higher.
+    pullRequest: 151
+    jiraIssue: 1330

```

Comment:
Internal. Do we want to keep it anyway? would not harm (if only the number is mentioned)...
But if we keep it as number we should rename the key to `jiraIssueNumber` (and `pullRequestNumber`.

Reply:
I would vote for keeping it as jiraIssueNumber.